mlflow/models/evaluation/base.py

```diff
@@ -456,11 +485,121 @@ def _start_run_or_reuse_active_run(run_id):
     else:
         if run_id and active_run.info.run_id != run_id:
             raise ValueError(
-                "An active run exists, you cannot specify another run_id when " "evaluating."
+                "An active run exists, you cannot specify another run_id when evaluating."
             )
         yield active_run.info.run_id
 
 
+def _normalize_evaluators_and_evaluator_config_args(
+        evaluators,
+        evaluator_config,
+):
+    from mlflow.models.evaluation.evaluator_registry import _model_evaluation_registry
+
+    def check_nesting_config_dict(_evaluator_name_list, _evaluator_name_to_conf_map):
+        return isinstance(_evaluator_name_to_conf_map, dict) and \
+               all(k in _evaluator_name_list and isinstance(v, dict)
+                   for k, v in _evaluator_name_to_conf_map.items())
+
+    if evaluators is None:
+        evaluator_name_list = list(_model_evaluation_registry._registry.keys())
+        if len(evaluator_name_list) > 1:
+            print(f'Hint: Multiple registered evaluators are found {evaluator_name_list} and '
+                  'they will all be used in evaluation. If you want to evaluate with one '
+                  'evaluator, specify the `evaluator` argument and (optional) specify the '
+                  '`evaluator_config` argument.')
```

Comment:
Nit: They might not all be used in evaluation if they do not support the specified model & model type.

```suggestion
            print(f'Multiple registered evaluators are found {evaluator_name_list} and '
                  'they will all be used in evaluation if they support the specified model type. If you want to evaluate with one '
                  'evaluator, specify the `evaluator` argument and (optional) specify the '
                  '`evaluator_config` argument.')
```